setup.py

```diff
@@ -52,12 +52,11 @@ def run(self):
         print("=> Building mod files ...")
 
         if platform.system() == 'Windows':
-            mod_path = op.join(op.dirname(__file__), 'mod')
             shell = True
         else:
-            mod_path = op.join(op.dirname(__file__), 'hnn_core', 'mod')
             shell = False
 
+        mod_path = op.join(op.dirname(__file__), 'hnn_core', 'mod')
         process = subprocess.Popen(['nrnivmodl'], cwd=mod_path,
                                    stdout=subprocess.PIPE,
                                    stderr=subprocess.PIPE, shell=shell)
```

Comment:
```suggestion
        process = subprocess.Popen(['nrnivmodl'], cwd=mod_dir,
                                   stdout=subprocess.PIPE,
                                   stderr=subprocess.PIPE, shell=shell)
```
...since `mod_dir` it's defined dynamically above

Reply:
Or maybe I'm misunderstanding what this does? Is this designed to build the nrn models in a potentially new directory (i.e., at `hnn_core/mod/`)?

Reply:
it changes into this directory: https://github.com/jonescompneurolab/hnn-core/tree/master/hnn_core/mod

and then runs `nrnivmodl` which compiles the mod files into the `dll` and `so` files

Reply:
Ah okay! Disregard my suggestion then.